diff -up mintlib-Git-20200504/string/strtoimax.c.orig mintlib-Git-20200504/string/strtoimax.c
--- mintlib-Git-20200504/string/strtoimax.c.orig	2022-07-10 13:51:22.101052785 +0200
+++ mintlib-Git-20200504/string/strtoimax.c	2022-07-10 13:51:15.641842103 +0200
@@ -21,6 +21,12 @@
 #include <inttypes.h>
 #include <stdlib.h>
 
+#ifdef __NO_INLINE__
+extern long long int __strtoll_internal (__const char *__restrict __nptr,
+                                              char **__restrict __endptr,
+                                              int __base, int __group);
+#endif
+
 intmax_t
 strtoimax (const char *__restrict nptr, char **__restrict endptr, int base)
 {
diff -up mintlib-Git-20200504/string/strtoumax.c.orig mintlib-Git-20200504/string/strtoumax.c
--- mintlib-Git-20200504/string/strtoumax.c.orig	2022-07-10 13:51:56.496849602 +0200
+++ mintlib-Git-20200504/string/strtoumax.c	2022-07-10 13:52:18.484162742 +0200
@@ -21,6 +21,12 @@
 #include <inttypes.h>
 #include <stdlib.h>
 
+#ifdef __NO_INLINE__
+extern unsigned long long int __strtoull_internal (__const char *__restrict __nptr,
+                                              char **__restrict __endptr,
+                                              int __base, int __group);
+#endif
+
 uintmax_t
 strtoumax (const char *__restrict nptr, char **__restrict endptr, int base)
 {
